pkg/bpf/map.go

```diff
@@ -167,6 +173,22 @@ func GetMapType(t MapType) MapType {
 	return t
 }
 
+// SetMapTypesFromProber initializes the supported map types from the given
+// prober. This function is useful for testing purposes, as we require
+// injecting our own mocked prober.
+func SetMapTypesFromProber(prober prober) {
```

Comment:
Why is this method exported if it's only being used from the test file which is in the same package?

Reply:
I figured it might be useful in the future, but I can unexport it for now. If it needs to be exported at some point, we can make that change.